airbyte-server/src/main/java/io/airbyte/server/converters/WebhookOperationConfigsConverter.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright (c) 2022 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.server.converters;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import io.airbyte.api.model.generated.WebhookConfigRead;
+import io.airbyte.api.model.generated.WebhookConfigWrite;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.config.WebhookConfig;
+import io.airbyte.config.WebhookOperationConfigs;
+import java.util.Collections;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+
+public class WebhookOperationConfigsConverter {
+
+  public static JsonNode toPersistenceWrite(List<WebhookConfigWrite> apiWebhookConfigs) {
+    if (apiWebhookConfigs == null) {
+      return Jsons.emptyObject();
+    }
+
+    final WebhookOperationConfigs configs = new WebhookOperationConfigs()
+        .withWebhookConfigs(apiWebhookConfigs.stream().map(WebhookOperationConfigsConverter::toPersistenceConfig).collect(Collectors.toList()));
+
+    return new ObjectMapper().convertValue(configs, JsonNode.class);
```

Comment:
i think we can definitely use some of the Jsons methods here: https://github.com/airbytehq/airbyte/blob/master/airbyte-commons/src/main/java/io/airbyte/commons/json/Jsons.java#L108